whitepapers/Tables.md

```diff
@@ -21,18 +24,18 @@ Identifier for a specific document from a specific source.
 | Column             | Type | Indexed | Nullable | FK  | Default | Description                                    |
 | :----------------- | :--: | :-----: | :------: | :-: | :-----: | :--------------------------------------------- |
 | `identifier`       | text |         |          |     |         | Identifier given to the document by the source |
-| `ingest_config_id` | int  |         |          |  ✓  |         | IngestConfig used to ingest the document       |
+| `ingest_config_id` | int  |    ✓    |          |  ✓  |         | IngestConfig used to ingest the document       |
```

Comment:
FK only implies an index in the case of django. There are also a decent amount of FKs that don't have indexes right now.